backend/tests/Squidex.Domain.Apps.Entities.Tests/Contents/Queries/ContentQueryServiceTests.cs

```diff
@@ -272,6 +311,26 @@ private Context CreateContext(bool isFrontend, bool allowSchema)
             return new Context(claimsPrincipal, Mocks.App(appId));
         }
 
+        private Context CreateContextWithOwnReadPermission(bool isFrontend, bool allowSchema)
```

Comment:
Can we not reuse the existing method and change allowSchema to an enum or so?

Reply:
How I undestood you mean instead CreateContextOwnPermission use only the method CreateContext that will has param enum(for example allowSchemaReadPermission, allowReadPermissionOwn). Correct?